### ISSUE TITLE
Importing physics colliders

### ISSUE THREAD

**Issue by Zemanzo** [Zemanzo/web-marbles#23]
**Description**
It should be possible to load models in the editor and use them as colliders (concave and convex).

**Minimum Acceptance Criteria**
- Support for convex colliders.
- Support for concave colliders.
- Models UI should be expanded to indicate support for these features per-model.

**Stretch goals**
- Support chunked terrain / height maps.
- Models that are used purely as colliders could have an optimized format in the level file.

**Is blocked by the following issues**
#70 #77
